src/test/java/uk/ac/ebi/subs/SampleTests.java

```diff
@@ -0,0 +1,111 @@
+package uk.ac.ebi.subs;
+
+import org.apache.http.Header;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.message.BasicHeader;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import uk.ac.ebi.subs.data.SampleResource;
+import uk.ac.ebi.subs.utils.TestJsonUtils;
+import uk.ac.ebi.subs.utils.TestUtils;
+
+import java.io.IOException;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertThat;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class SampleTests {
+
+    static String submitterEmail = "api-tester@ebi.ac.uk";
+    static String teamName = "api-tester";
+
+    static String submissionsApiBaseUrl = "http://submission-dev.ebi.ac.uk/api/submissions/";
```

Comment:
Would be neat if we could get these properties from application.properties. Then could test against local/qa envs.